eks-anywhere-common/Testers/Pulumi/.gitignore

```diff
@@ -0,0 +1 @@
+kubeconfig.yml
```

Comment:
Please remove this file.

Reply:
Do you mean the `.gitignore` file?

Reply:
Why do we need this `.gitignore` file. We have one at root.

Reply:
Done